.env.example

```
@@ -51,17 +51,60 @@ NEXT_PUBLIC_PEG_ASSET=Gold
 # Default: Auto-generated based on PEG_UNIT and PEG_ASSET
 #   - For \"1g\" + \"Gold\": \"Pegged to 1g of gold.\"
 #   - For \"1 USD\" + \"USD\": \"Pegged to 1 USD.\" (smart detection)
 # Example alternatives: \"Pegged to 1 US Dollar.\", \"Pegged to 1 ounce of silver.\"
 # NEXT_PUBLIC_PEG_DESCRIPTION=Pegged to 1 gram of gold.
 
+# Theme colors (optional - fallback to Gold defaults if not set)
+# All values must be valid hex colors.
+# Default primary/glow/stable: #ffd007
+# Default volatile: #ff3b3b
+NEXT_PUBLIC_THEME_PRIMARY="#ffd007"
+NEXT_PUBLIC_THEME_GLOW="#ffd007"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_THEME_GLOW key should go before the NEXT_PUBLIC_THEME_PRIMARY key

(UnorderedKey)